internal/middlewares/access_managment.go

```diff
@@ -0,0 +1,136 @@
+package middlewares
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/golang-jwt/jwt"
+
+	"git.epam.com/epm-lstr/epm-lstr-lc/be/internal/authentication"
+	"git.epam.com/epm-lstr/epm-lstr-lc/be/internal/utils"
+)
+
+type AccessManager struct {
+	endpoints       existingEndpoints
+	acceptableRoles []string
+	fullAccessRoles []string
+	accessMap       map[string]existingEndpoints
```

Comment:
Should we use blacklist or whitelist logic for access management? I think a blacklist is better because that minimizes the chance of making some sensitive endpoint open.

Reply:
Blacklist is OK.